### DISCUSSION TITLE
窗口最大化功能

### DISCUSSION THREAD

**Discussion by sanweiya** [Meloong-Git/PCL#2665] [讨论]
笔记本小屏幕比较局促，查找梳理mod时需要把窗口拖大才舒服，但是平时也不希望启动器占用太多屏幕空间，能加上窗口最大化按钮就好了

**Comment by shanlisi567** [comment]
我建议任务管理器

**Comment by iPodtouch6557** [comment]
印象中是拒绝的，就是懒得搜.jpg，有知道的可以补充
~-口X的文艺复兴~

**Reply by MoYuan-CN** [reply]
印象里被拒绝了 但是我找不到 Issue 了（悲

**Comment by kong-que-dnf123** [comment]
> 印象里被拒绝了 但是我找不到 Issue 了（悲

你说的不会是 #1161 的最小化功能吧......

**Reply by iPodtouch6557** [reply]
记得是还有个最大化

**Reply by MoYuan-CN** [reply]
不是 #1161

**Reply by SuoHouDaiShi** [reply]
#90 (#2745 )
@JingHai-Lingyun a.a

**Reply by MoYuan-CN** [reply]
啊 你找到了
